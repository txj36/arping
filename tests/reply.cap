ARP, Reply 192.0.2.100 is-at 00:01:02:33:44:00, length 44
	0x0000:  0001 0800 0604 0002 0001 0233 4400 c000
	0x0010:  0264 ffff ffff ffff c000 0265 0000 0000
	0x0020:  0000 0000 0000 0000 0000 0000
