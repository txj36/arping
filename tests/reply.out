ARPING 192[.]0[.]2[.]101
Sending ARP reply

--- 192[.]0[.]2[.]101 statistics ---
1 packets transmitted, 0 packets received, 100% unanswered \(0 extra\)
